avm/res/desktop-virtualization/scaling-plan/main.bicep

```
@@ -0,0 +1,227 @@
+metadata name = 'Azure Virtual Desktop Scaling Plan'
+metadata description = 'This module deploys an Azure Virtual Desktop Scaling Plan.'
+metadata owner = 'Azure/module-maintainers'
+
+@sys.description('Required. Name of the scaling plan.')
+param name string
+
+@sys.description('Optional. Location of the scaling plan. Defaults to resource group location.')
+param location string = resourceGroup().location
+
+@sys.description('Optional. Friendly name of the scaling plan.')
+param friendlyName string = name
+
+@sys.description('Optional. Time zone of the scaling plan. Defaults to UTC.')
+param timeZone string = 'UTC'
+
+@sys.description('Optional. Host pool type of the scaling plan.')
+param hostPoolType string = 'Pooled'
+
+@sys.description('Optional. Exclusion tag to be used for exclusion of VMs from scaling plan.')
+param exclusionTag string = ''
+
+@sys.description('Optional. Schedules of the scaling plan.')
+param schedules array = []
+
+@sys.description('Optional. Host pool references of the scaling plan.')
+param hostPoolReferences array = []
+
+@sys.description('Optional. Description of the scaling plan.')
+param description string = name
+
+@sys.description('Optional. Tags of the scaling plan.')
```

```suggestion
@sys.description('Optional. Tags of the resource.')
```
Suggestion as per [specs](https://azure.github.io/Azure-Verified-Modules/specs/shared/interfaces/).

---

Note: Requires regeneration of both the readme & json